docs/standard/glossary.md

```diff
@@ -270,6 +270,16 @@ Legacy .NET documentation sometimes uses ".NET platform" to mean either an [impl
 
 "Platform" has a different meaning in the phrase "developer platform," which refers to software that provides tools and libraries for building and running apps. .NET is a cross-platform, open-source developer platform for building many different types of applications.
 
+## poco
```

Comment:
I think poco should be POCO everywhere in this section.

Reply:
Even in the heading?